articles/resources/relnotes.md

```diff
@@ -17,6 +17,23 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.12.20092342
+
+*Release date: September 29th, 2020*
+
+This release contains the following:
+
+- Performance improvements for simulator: better application of gate fusion decisions; improved parallelization on Linux system; added intelligent scheduling of gate execution; bug fixes; overall speed improvement of ~5x (even faster on larger problems).
+- IntelliSense features are now supported for Q# files in Visual Studio and Visual Studio Code even without a project file.
+- Various Q#/Python interoperability improvements and bug fixes, including better support for NumPy data types.
+- Improvements to the Microsoft.Quantum.Arrays namespace (microsoft/QuantumLibraries#313)
+- Added a new language repo (QIR blog)[TODO]
+- General bug fixes.
```

Comment:
Is "general bug fixes" across all repos? We could tag the relevant GitHub issue numbers, happy to help with this.